hw/ip/otbn/data/bignum-insns.yml

```diff
@@ -40,7 +40,7 @@
     b = UInt(wrs2)
 
     fg = DecodeFlagGroup(flag_group)
-    sb = UInt(shift_bytes)
+    sb = 8 * UInt(shift_bits)
     st = DecodeShiftType(shift_type)
   operation: |
     b_shifted = ShiftReg(b, st, sb)
```

Comment:
Can you also update the pseudo-code of `ShiftReg()` now that sb is bits instead of bytes?

Reply:
Oh, wait! I've got that backwards (I put the same bug in the implementation then fixed it, but forgot this). I'll change this line to `UInt(shift_bits) // 8`, then `sb` has the same meaning as before. I wanted to keep it that way to make it clear that `ShiftReg` didn't need to be a full 256-bit shifter.